fs_attachment/models/fs_storage.py

```diff
@@ -1,6 +1,7 @@
 # Copyright 2023 ACSONE SA/NV
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
+from __future__ import annotations
```

Comment:
By default, all oca addons are tested with py 3.10 and odoo 16 itself is tested on py 3.10. You  should use py 3.10

Reply:
yeah - but odoo 16 does also work with python 3.9 - even the default docker Dockerfile does use debian bullseye - which is based on python 3.9.
And other modules in the repo do already use this future import - so it should be consistent across the modules.